types/bun/globals.d.ts

```diff
@@ -655,7 +669,7 @@ declare class Request implements BlobInterface {
 }
 
 interface Crypto {
-  getRandomValues(array: TypedArray): void;
+  getRandomValues<T = TypedArray>(array: T): T;
```

Comment:
T should also extend TypedArray (we shouldn't allow something like `string`).
```suggestion
  getRandomValues<T extends TypedArray = TypedArray>(array: T): T;
```